controllers/contacts/updateStatus.js

```diff
@@ -0,0 +1,28 @@
+import { updateContact } from "#service/index.js";
+
+async function updateStatus(req, res, next) {
+  const { contactId } = req.params;
+  const { favorite = false } = req.body;
```

Comment:
Tutaj trzeba byłoby zrobić taką walidację
```
Jeżeli body nie ma, zwraca json z kluczem {"message": "missing field favorite"} i statusem 400.
```

Reply:
Ja bym nie zakładał że to jest false, ale sprawdził czy to istnieje jeśli nie istnieje to trzeba to poinformować bo my oczekujemy że to przyjdzie Jeżeli ktoś wchodzi na ten endpoint.

Reply:
Poprawione